code/game/objects/structures/dropship_ammo.dm

```diff
@@ -414,3 +406,46 @@
 	T.visible_message("<span class='warning'>You see a tiny flash, and then a blindingly bright light from a flare as it lights off in the sky!</span>")
 	playsound(T, 'sound/weapons/guns/fire/flare.ogg', 50, 1, 4) // stolen from the mortar i'm not even sorry
 	QDEL_IN(src, rand(70 SECONDS, 90 SECONDS)) // About the same burn time as a flare, considering it requires it's own CAS run.
+
+//Railgun, currently the same as GAU, but being seperated so that changes to the GAU don't affect the railgun
+/obj/structure/ship_ammo/railgun
+	name = "Railgun Ammo"
+	desc = "This is not meant to exist"
+	travelling_time = 6 SECONDS
+	ammo_count = 400
+	max_ammo_count = 400
+	ammo_used_per_firing = 40
+	var/bullet_spread_range = 5
+	var/attack_width = 3
+	travelling_time = 6 SECONDS
+
+/obj/structure/ship_ammo/railgun/detonate_on(turf/impact, attackdir = NORTH)
+	var/turf/beginning = impact
+	var/revdir = REVERSE_DIR(attackdir)
+	for(var/i=0 to bullet_spread_range)
+		beginning = get_step(beginning, revdir)
+	var/list/strafelist = list(beginning)
+	strafelist += get_step(beginning, turn(attackdir, 90))
```

Comment:
What is this copypaste?

Reply:
Splinter asked to sperate the railgun from GAU so that GAU changes don't affect the railgun. If preferred I can change back to it being a subtype of GAU ammo but if the GAU were to get any changes the railgun would be affected again

Reply:
Make it a child for now this copypaste is nasty